modules/security-jwt/src/main/java/com/devonfw/module/security/jwt/util/TokenConfigProperties.java

```diff
@@ -0,0 +1,249 @@
+package com.devonfw.module.security.jwt.util;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * This class contain properties related to {@link TokenAuthenticationUtil}
+ *
+ * @since 3.2.0
+ */
+@Configuration
+@ConfigurationProperties(prefix = "security.authentication.token")
+public class TokenConfigProperties {
```

Comment:
This class does not define any defaults. So we should follow the `convention-over-configuration` approach and define defaults where suitable (e.g. expiration, alias for private/public key, could have reasonable defaults. Some other properties might not have defaults such as `issuer` - or should we use `devonfw` as default?).